infrastructure/resource_group_level/modules/webapps/trade-finance-manager-api-calculated-variables.bicep

```diff
@@ -36,33 +37,27 @@ resource cosmosDbAccount 'Microsoft.DocumentDB/databaseAccounts@2023-04-15' exis
 // Then there are the calculated values. 
 var mongoDbConnectionString = replace(cosmosDbAccount.listConnectionStrings().connectionStrings[0].connectionString, '&replicaSet=globaldb', '')
 
-var connectionStringsCalculated = {  
-  MONGO_INITDB_DATABASE: {
-    type: 'Custom'
-    value: cosmosDbDatabaseName
-  }
-  MONGODB_URI: {
-    type: 'Custom'
-    value: mongoDbConnectionString
-  }
-  AZURE_NUMBER_GENERATOR_FUNCTION_URL: {
-    type: 'Custom'
-    value: 'https://${numberGeneratorFunctionDefaultHostName}'
-  }
-  TFM_UI_URL: {
-    type: 'Custom'
-    value: tfmUiUrl
-  }
-} 
+var settingsCalculated = {  
```

Comment:
can this be calculatedAppSettings please just so it's the same naming as appSettings elsewhere